.github/workflows/create-prerelease-branch.yml

```diff
@@ -144,8 +144,8 @@ jobs:
           git checkout -b groom-changelog-${{github.event.inputs.bumptype}}/${{ steps.version.outputs.version }}
           echo "groombranch=groom-changelog-${{github.event.inputs.bumptype}}/${{ steps.version.outputs.version }}" >> $GITHUB_OUTPUT
 
-      - name: Add trivial changefile so CI does not complain about the CHANGELOG grooming PR
-        run: node common/scripts/changelog/change.mjs -m 'Bumped versions for release' --type none
+      # - name: Add trivial changefile so CI does not complain about the CHANGELOG grooming PR
+      #   run: node common/scripts/changelog/change.mjs -m 'Bumped versions for release' --type none
```

Comment:
I guess we want this so that the next release driver doesn't hit that error when generating the changelogs?

Reply:
We had to disable this because we were unable to run this in automation. this changelog needs to be made manually now with the release driver since the beachball can't take cli commands now. 

I will update it so there is a step listed that is manual for the release driver and update the release notes so that they can see they need to do this themselves